stdlib/http/src/main/java/org/ballerinalang/net/http/actions/websocketconnector/PushText.java

```diff
@@ -55,10 +52,10 @@ public static Object externPushText(Strand strand, ObjectValue wsConnection, Str
                     .getNativeData(WebSocketConstants.NATIVE_DATA_WEBSOCKET_CONNECTION_INFO);
             ChannelFuture future = connectionInfo.getWebSocketConnection().pushText(text, finalFrame);
             WebSocketUtil.handleWebSocketCallback(callback, future, log);
+            callback.setReturnValues(null);
```

Comment:
Does this return value setter required? Isn't is handled by handleWebSocketCallback() method?